SpaceCadetPinball/TPlunger.cpp

```diff
@@ -109,7 +109,7 @@ int TPlunger::Message(int code, float value)
 			if (PullbackTimer_)
 				timer::kill(PullbackTimer_);
 			PullbackTimer_ = 0;
-			if (code == 1005)
+			if (code == 1005) // intended that this can never be reached (same as original)
```

Comment:
Original has cases collapsed with goto:

case 1005:
case 1009:
case 1010:
	goto LABEL_12;
…
case 1024:
	if ( this->BallFeedTimer )
		timer_kill(this->BallFeedTimer);
	this->BallFeedTimer = 0;
LABEL_12:

I will reshuffle this into
case 1005:
case 1009:
case 1010:
case 1024:
if(case == 1024)
	…